traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -24,6 +24,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/topology"
```

Comment:
this guy should be down below with the others

Reply:
Relegated guy in d04cfd8c76